interpreter/simplecompositevalue.go

```diff
@@ -151,6 +151,15 @@ func (v *SimpleCompositeValue) GetMember(context MemberAccessibleContext, locati
 	return nil
 }
 
+func (v *SimpleCompositeValue) GetMethod(
+	context MemberAccessibleContext,
+	locationRange LocationRange,
+	name string,
+) FunctionValue {
+	// TODO:
```

Comment:
I guess `SimpleCompositeValue.Fields` should also be split into fields and methods, there might be some uses of `SimpleCompositeValue` where methods are stored in `Fields`

Reply:
Yes. Will open a follow-up PR, as it will need some refactoring for many builtin values, and this PR is already big.

Reply:
Added in https://github.com/onflow/cadence/pull/3890